lib/booktype/apps/core/context_processors.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings as django_settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+# Inspired by https://github.com/mfogel/django-settings-context-processor
+def settings(request):
```

Comment:
I would change this to something like `settings_variables` because it could look a bit confusing and to avoid shadowing of django settings.